overlord/snapstate/storehelpers.go

```diff
@@ -201,7 +201,7 @@ var installSize = func(st *state.State, snaps []minimalInstallInfo, userID int)
 	return total, nil
 }
 
-func installInfo(ctx context.Context, st *state.State, name string, revOpts *RevisionOptions, userID int, flags Flags, deviceCtx DeviceContext) (store.SnapActionResult, error) {
+func installInfo(ctx context.Context, st *state.State, name string, revOpts *RevisionOptions, userID int, deviceCtx DeviceContext) (store.SnapActionResult, error) {
```

Comment:
Please don't drop Flags here, we have a TODO below about IgnoreValidation which is precisely what flags are about (and I'm actually going to address it very soon).

Reply:
Oh no problem, I'll revert it